sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveSQLViewSuite.scala

```diff
@@ -137,4 +137,24 @@ class HiveSQLViewSuite extends SQLViewSuite with TestHiveSingleton {
       }
     }
   }
+
+  test("Add HiveVoidType to compatible with Hive void type") {
```

Comment:
let's add the JIRA ticket ID in the beginning.

Reply:
added.